master/buildbot/steps/artifact.py

```diff
@@ -298,6 +362,13 @@ def rsyncWithRetry(step, origin, destination, port=None):
 
     return retryCommandLinuxOS(rsync_command)
 
+def mkDir(step):
+    if _isWindowsSlave(step):
+        return r'C:\cygwin64\bin\mkdir.exe'
```

Comment:
@mariangemarcano @opdude @campos-ddc 
The problem with mkdir is that it is one of the commands included into the command.com. it means that it does not matter what are you going to add into the PATH it will be executed first
`createDir` in global.config does it the same way. I'll try to invent something wich does not require hard-coding it in this part of the code